backend/src/bundles/oauth/config/oauth-config.ts

```diff
@@ -33,6 +33,20 @@ const oauthConfigurations = {
         },
         scope: ['public_profile', 'email'],
     },
+    linkedin: {
+        credentials: {
+            client: {
+                id: config.ENV.OAUTH.LINKEDIN.CLIENT_LINKEDIN_ID,
+                secret: config.ENV.OAUTH.LINKEDIN.CLIENT_LINKEDIN_SECRET,
+            },
+            auth: oauthPlugin.LINKEDIN_CONFIGURATION,
+            options: {
+                bodyFormat: 'form' as const,
```

Comment:
Is there a way to avoid using `as`?

Reply:
Corrected:
` options: {
                bodyFormat: config.ENV.OAUTH.LINKEDIN.OPTIONS.bodyFormat,
                authorizationMethod:
                    config.ENV.OAUTH.LINKEDIN.OPTIONS.authorizationMethod,
            },`